weblab/templates/entities/entity_newversion.html

```diff
@@ -6,7 +6,7 @@
 {% block content %}
   {% include "./includes/entity_header.html" %}
 
-  <h2>New version</h2>
+  <h2>Create a new version</h2>
```

Comment:
Can we have the entity type in there as well?

Reply:
As in 'Create a new model version' or 'Create a new version of this model'?

Reply:
Yes please!